src/mlpack/methods/ann/loss_functions/dice_loss.hpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file dice_loss.hpp
+ * @author N Rajiv Vaidyanathan
+ *
+ * Definition of the dice loss function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LOSS_FUNCTIONS_DICE_LOSS_HPP
+#define MLPACK_METHODS_ANN_LOSS_FUNCTIONS_DICE_LOSS_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The dice loss performance function measures the network's
+ * performance according to the dice coefficient
+ * between the input and target distributions.
+ *
+ * For more information see the following.
+ *
+ * @inproceedings{milletari2016v,
+ *   title = {V-net: Fully convolutional neural networks
+ *            for volumetric medical image segmentation},
+ *   author = {Milletari, Fausto and Navab, Nassir and Ahmadi, Seyed-Ahmad},
+ *   booktitle = {3D Vision (3DV), 2016 Fourth International Conference on},
+ *   pages = {565--571},
```

Comment:
Hmm, not sure if two dashes have to be here.

Reply:
I have replaced the Google Scholar version with the `arxiv BibTex` version as suggested by @zoq . Please let me know if I should keep it or return back to the Google Scholar version.